### PR TITLE
Add README and gitignore

### PR DESCRIPTION
**Pull request by lofidevops**
Some simple additions for first-timers like me :)

### CODE REVIEW

**Review by maxiberta — APPROVED** [review]
Nice, thanks! Might I suggest adding a `make bootstrap` target to automatically install all deps (i.e. install the doc builder snap iiuc), and update README accordintly?
Also (I was not familiar with this project, sorry): I noticed the temp build directory is at the top-level of the user's home (`BUILDDIR=$HOME/snapstore-snap-docs`), which I find a bit unexpected. It'd be great if it were under the current directory, or at least under `$HOME/.cache`. But, feel free to ignore this. We can always improve things down the road.

**Comment by lofidevops** [comment]
@maxiberta Great suggestion on the bootstrap. :+1: 

When I run `make` it appears in the local subfolder as expected, despite the reference to $HOME. I'm guessing this is used in production so I'll open a separate ticket to discuss the best course.

**Comment by maxiberta** [comment]
> When I run make it appears in the local subfolder as expected, despite the reference to $HOME. I'm guessing this is used in production so I'll open a separate ticket to discuss the best course.

Hm that's weird. It used $HOME in my tests:

```sh
$ make
./build.sh
Built:
- /home/max/snapstore-snap-docs/html/en/api-overrides.html
- /home/max/snapstore-snap-docs/html/en/migrate-beta.html
- /home/max/snapstore-snap-docs/html/en/register.html
- /home/max/snapstore-snap-docs/html/en/index.html
- /home/max/snapstore-snap-docs/html/en/install.html
- /home/max/snapstore-snap-docs/html/en/devices.html
- /home/max/snapstore-snap-docs/html/en/airgap.html
- /home/max/snapstore-snap-docs/html/en/trouble.html
- /home/max/snapstore-snap-docs/html/en/api-authentication.html
- /home/max/snapstore-snap-docs/html/en/https.html
- /home/max/snapstore-snap-docs/html/en/overrides.html
- /home/max/snapstore-snap-docs/html/en/airgap-device-auth.html
Copied /home/max/snapstore-snap-docs/media to /home/max/snapstore-snap-docs/html/media
$ 
```

**Comment by lofidevops** [comment]
@maxiberta
* Added a `make bootstrap` as requested.
* Created https://github.com/canonical-ols/snapstore-snap-docs/issues/31 for the $HOME issue

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,6 @@
+bootstrap:
+	snap install documentation-builder
```
Comment (maxiberta) [inline Makefile:2]: Should this be prefixed by `sudo`?

**Review by maxiberta — APPROVED** [review]
LGTM, thanks!

**Pull request merged**